app/assets/stylesheets/components/_banner.scss

```diff
@@ -20,17 +20,8 @@
   }
 }
 
-.app-c-banner__main {
-  @include grid-column( 2 / 3, $full-width: tablet, $float: right );
-
-  @include media(tablet) {
-    padding-right: 0;
-    border-left: 1px solid $white;
-  }
-}
-
 .app-c-banner__aside {
-  margin-top: $gutter-half;
+  padding-top: $gutter-one-third;
 
   @include media(tablet) {
     margin-top: 0;
```

Comment:
I'm not sure these styles are needed now that the aside isn't located to the left of the other banner content. Probably worth removing and testing to make sure it still works on all screen sizes

Reply:
Sounds good!